kontroller/qml/ManageZones.qml

```diff
@@ -18,7 +18,7 @@ Page {
             anchors.right: parent.right
             anchors.leftMargin: Theme.horizontalPageMargin
             anchors.rightMargin: Theme.horizontalPageMargin
-            text:qsTr("To learn new zones, start polling and then switch the audio output inside Kodi. Zones will be added automatically. Don’t forget go back and save afterwhile")
+            text:qsTr("To learn new zones, start polling and then switch the audio output inside Kodi to add them automatically. Go back and save after a while.")
```

Comment:
Not 100% sure what happens here.

Reply:
This is a work in progress. At a time, i had a try to implement multizone support via selecting the audio output of kodi (using pulseaudio to create a lot of virtual devices), speakers were powered from the same source device but in different rooms. Due to no longer needing this, this is now disabled in the app (or should be). This may come up again, but in that case it will need some changes.